services/operator_info.go

```diff
@@ -0,0 +1,134 @@
+package services
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Rocket-Rescue-Node/credentials"
+	"github.com/Rocket-Rescue-Node/credentials/pb"
+	"github.com/Rocket-Rescue-Node/rescue-api/models"
+	authz "github.com/Rocket-Rescue-Node/rescue-api/models/authorization"
+	"github.com/Rocket-Rescue-Node/rescue-api/util"
+	"go.uber.org/zap"
+)
+
+const (
+	// The maximum age for a operator info request to be considered valid.
+	operatorInfoRequestMaxAge = time.Duration(15) * time.Minute
+)
+
+type OperatorInfo struct {
+	CredentialEvents []int64 `json:"credentialEvents"`
+	QuotaSettings    *Quota  `json:"quotaSettings,omitempty"`
+}
+
+func CreateOperatorInfo(credentialEvents []int64, quotaSettings Quota) (*OperatorInfo, error) {
+	message := OperatorInfo{}
+	message.CredentialEvents = credentialEvents
+	message.QuotaSettings = &quotaSettings
+
+	return &message, nil
```

Comment:
if you do decide to keep the function for whatever reason, at least remove the `error` return type, which is known-nil at compiletime.

Reply:
Function removed.